### PR TITLE
Issue #ISAICP-2316: User can edit profile.

### PR DESCRIPTION
**Pull request by sandervd**

### CODE REVIEW

File: web/profiles/joinup/joinup.info.yml [hunk 1]
```diff
@@ -110,23 +108,13 @@ config_devel::
...
-  - field.storage.node.body
-  - field.storage.node.field_additional_address_info
-  - field.storage.node.field_agenda
-  - field.storage.node.field_contact_email
-  - field.storage.node.field_end_date
-  - field.storage.node.field_logo
-  - field.storage.node.field_short_title
-  - field.storage.node.field_start_date
-  - field.storage.node.field_web_url
-  - field.storage.node.og_group_ref
```
Comment (pfrenssen) [inline web/profiles/joinup/joinup.info.yml]: Why are these removed? Does not look related to this ticket?

Comment (sandervd) [inline web/profiles/joinup/joinup.info.yml]: These config entities are moved elsewhere. I just cleanup up the references.

**Comment by pfrenssen** [comment]
Cheers!
